src/DatePicker/DatePickerInput.svelte

```diff
@@ -117,7 +117,7 @@
       class:bx--date-picker__input="{true}"
       class:bx--date-picker__input--invalid="{invalid}"
       class:bx--date-picker__input--sm="{size === 'sm'}"
-      class:bx--date-picker__input--xl="{size === 'xl'}"
+      class:bx--date-picker__input--lg="{size === 'lg' || size === 'xl'}"
```

Comment:
@theetrain I wonder if we should keep supporting legacy size `xl` for a while. What do you think?